gsi/gss_assist/source/programs/grid-mapfile-add-entry.txt

```diff
@@ -65,6 +65,9 @@ The full set of command-line options to *grid-mapfile-add-entry* are:
 *-mapfile 'MAPFILE', -f 'MAPFILE'*::
     Modify the gridmap file named by 'MAPFILE' instead of the default.
 
+*-force::
```

Comment:
Now I see the problem: A `*` is missing after `*-force`, then the manpage output should be created correctly.

Reply:
Changes applied.